src/components/scrim/edit-scrim-form.tsx

```diff
@@ -205,11 +217,55 @@ export function EditScrimForm({
                 <FormDescription>
                   This is the team that will be associated with this scrim. You
                   can assign scrims to teams that you manage or own.
-                </FormDescription>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
+              </FormDescription>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        
+        <FormField
```

Comment:
Your indentation is wrong, please make sure to format with Prettier before submitting.